content/applications/sales/crm/track_leads/unattended_leads_report.rst

```diff
@@ -0,0 +1,180 @@
+   *Activities* feature in the *chatter*
+
+   For more information, refer to :doc:`Activities <../../../essentials/activities>`
+
+Create an unattended leads report
+=================================
+
+To create an unattended leads report, first navigate to :menuselection:`CRM app --> Reporting -->
+Pipeline` to open the :guilabel:`Pipeline Analysis` dashboard. Click into the :guilabel:`Search...`
+bar at the top of the page, and remove all of the default filters.
+
+.. note::
+   The :guilabel:`Created on` filter can remain active, as this variable may be useful to include in
+   the report.
+
+Next, add custom filters by clicking the :guilabel:`🔻(triangle pointed down)` icon to the right of
+the :guilabel:`Search...` bar to open the drop-down menu that contains :guilabel:`Filters`,
+:guilabel:`Group By`, and :guilabel:`Favorites` columns. Under the :guilabel:`Filters` column,
+click :guilabel:`Add Custom Filter`, which opens an :guilabel:`Add Custom Filter` pop-up window.
+
+The :guilabel:`Add Custom Filter` pop-up window allows for the creation of more specific filters.
+
+Add filter for past due activities
+----------------------------------
+
+Click the first field for the new rule, and type `Activities` in the :guilabel:`Search...` bar, or
+scroll to search through the list to locate it. Then, next to :guilabel:`Activities`, click the
+:guilabel:`> (greater than sign)` to open a new drop-down menu with secondary conditions.
+
+Type `Due Date` in the :guilabel:`Search...` bar, or scroll to search through the list. Click
+:guilabel:`Due Date` to add it to the rule.
+
+  .. image:: unattended_leads_report/activities-due.png
+     :align: center
+     :alt: Custom filter pop-up with emphasis on the options for activities and due date.
+
+Then, click into then next field and select :guilabel:`<=` from the drop-down menu. Selecting this
+operator includes all activities with a due date up to, and including, the date selected in the next
+field.
+
+The third field can be left as today's date, or adjusted as needed.
+
+Exclude unassigned leads
+------------------------
+
+Click :guilabel:`New Rule`. Click the first field for the new rule, and type `Salesperson` in the
+:guilabel:`Search...` bar, or scroll to search through the list to locate it.
+
+Click in the rule's second field, then select :guilabel:`is set` from the drop-down menu. Selecting
+this operator excludes any leads not assigned to a specific salesperson.
+
+Add a Sales team
+----------------
+
+Click :guilabel:`New Rule`. Click the first field for the new rule, and type `Sales Team` in the
+:guilabel:`Search...` bar, or scroll to search through the list to locate it.
+
+Click in the rule's second field, select :guilabel:`is in` from the drop-down menu. Selecting this
+operator limits results to the sales teams selected in the next field.
+
+In the third field, select the desired sales team from the drop-down menu. Multiple teams can be
+selected in this field.
```

Comment:
This is cool 👍 I didn't know you could add multiple teams here and it would count the parameters as part of an OR argument. I thought you had to create an **ANY** branch with 2+ choices in order to do this.

Suggestion to specify further what adding multiple teams in this field means, logically:
```suggestion
In the third field, select the desired sales team from the drop-down menu. Multiple teams can be
added in this field, where each parameter will be treated with an "or" (e.g. "any") operator in the search logic.
```